cmd/manager/main.go

```diff
@@ -171,3 +185,42 @@ func main() {
 		klog.Fatalf("Error starting manager: %v", err)
 	}
 }
+
+// newConfigManagedClient returns a controller-runtime client that can be used to access the openshift-config-managed
+// namespace.
+func newConfigManagedClient(config *rest.Config) (runtimeclient.Client, manager.Runnable, error) {
+	scheme := runtime.NewScheme()
+	if err := corev1.AddToScheme(scheme); err != nil {
+		return nil, nil, err
+	}
```

Comment:
This is a blank scheme that only has core types right? Wondering if it might be better to share the mgr scheme here? Then it will have all types registered? May prevent future issues if we try to get a non-core type from the config-managed namespace 🤔

Reply:
Sounds good. I'll re-use the scheme and the REST mapper from the manager. I'll have to add the core types to that scheme.